CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ if (BUILD_C_PLUGINS)
 	if (Modbus)
 		add_subdirectory(C/plugins/south/modbus)
 	endif()
-	add_subdirectory(C/plugins/north/omf)
+	add_subdirectory(C/plugins/north/PI_Server)
```

Comment:
I'm not sure about the convention followed here. I would sugegst to name it `PiServer` or `pi-server`

Reply:
this was asked/defined with Mark.

Reply:
Then its fine.